### PR TITLE
DEVDOCS-4831: [update] getImageManagerImageSrcset

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-4831]
{Ticket number or summary of work}

## What changed?
Updated the description for getImageManagerImageSrcset

## Anything else?
Add related PRs, salient notes, ticket numbers, etc.

ping {names}

[DEVDOCS-4831]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4831?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.mdx [hunk 1]
```diff
@@ -757,11 +757,11 @@ Returns an [Image Manager](https://support.bigcommerce.com/s/article/Using-the-I
 ```handlebars showLineNumbers
 {{getImageManagerImageSrcset "asset.jpg"}}
 <!-- =>
-https://cdn.bcapp/3dsf74g/images/stencil/80w/image-manager/asset.jpg 80w, https://cdn.bcapp/3dsf74g/images/stencil/160w/image-manager/asset.jpg 160w, https://cdn.bcapp/3dsf74g/images/stencil/320w/image-manager/asset.jpg 320w, https://cdn.bcapp/3dsf74g/images/stencil/640w/image-manager/asset.jpg 640w, https://cdn.bcapp/3dsf74g/images/stencil/960w/image-manager/asset.jpg 960w, https://cdn.bcapp/3dsf74g/images/stencil/1280w/image-manager/asset.jpg 1280w, https://cdn.bcapp/3dsf74g/images/stencil/1920w/image-manager/asset.jpg 1920w, https://cdn.bcapp/3dsf74g/images/stencil/2560w/image-manager/asset.jpg 2560w -->
```
Comment (jairo-bc) [inline docs/stencil-docs/reference-docs/handlebars-helpers-reference.mdx:760]: Widths are still returned, that shouldn't be changed. 
Take a look here https://github.com/bigcommerce/paper-handlebars/blob/master/spec/helpers/getImageManagerImageSrcset.js#L39

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.mdx [hunk 2]
```diff
@@ -757,11 +757,11 @@ Returns an [Image Manager](https://support.bigcommerce.com/s/article/Using-the-I
...
 
-{{getImageManagerImageSrcset "folder/asset.jpg" width=123}}
+{{getImageManagerImageSrcset "folder/asset.jpg"}}
  <!-- =>
-https://cdn.bcapp/3dsf74g/images/stencil/80w/image-manager/folder/asset.jpg 80w, https://cdn.bcapp/3dsf74g/images/stencil/160w/image-manager/folder/asset.jpg 160w, https://cdn.bcapp/3dsf74g/images/stencil/320w/image-manager/folder/asset.jpg 320w, https://cdn.bcapp/3dsf74g/images/stencil/640w/image-manager/folder/asset.jpg 640w, https://cdn.bcapp/3dsf74g/images/stencil/960w/image-manager/folder/asset.jpg 960w, https://cdn.bcapp/3dsf74g/images/stencil/1280w/image-manager/folder/asset.jpg 1280w, https://cdn.bcapp/3dsf74g/images/stencil/1920w/image-manager/folder/asset.jpg 1920w, https://cdn.bcapp/3dsf74g/images/stencil/2560w/image-manager/folder/asset.jpg 2560w -->
```
Comment (jairo-bc) [inline docs/stencil-docs/reference-docs/handlebars-helpers-reference.mdx:764]: Here also it should remain

**Review by jairo-bc — DISMISSED** [review]

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.mdx (hunk 2 above)
Comment (bc-traciporter) [reply docs/stencil-docs/reference-docs/handlebars-helpers-reference.mdx:764]: Oh okay, thank you @jairo-bc 
The widths are not in the request but they appear in the response.

**Review by bc-andreadao — APPROVED** [review]

**Comment by bc-traciporter** [comment]
@jairo-bc can you approve the changes? I am still not able to merge the PR.

**Pull request merged**